### ISSUE TITLE
soketi seems died. doesn't working with node 21

### ISSUE THREAD

**Issue by lei-cao** [renchris/replicache-demo-with-nextjs#3]
https://github.com/soketi/soketi

**Comment by renchris** [comment]
Yes, Soketi works up with Node 18 LTS. If you wish to use higher versions of node you need to fork Soketi and simply update the `uWebSockets.js` version in the `package.json` file as so here: https://github.com/soketi/soketi/pull/1074

Also, from my understanding, you can have the `soketi start` Soketi server running in an environment using up to Node 18 LTS separately from your main repository running in an environment on a version higher than Node 18 LTS ie. Node 21

**Comment by lei-cao** [comment]
thank you for the guide & support
